### PR TITLE
feature/96/faq

### PR DESCRIPTION
**Pull request by norachams**
Screenshot 2024-03-03 at 2 42 44 AM
Screenshot 2024-03-03 at 2 42 27 AM

### CODE REVIEW

File: src/components/sections/FAQsection.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <div className="h-full flex flex-col justify-center items-center">
+      <h2 className="mt-60 text-center text-black font-bold">
+        FAQ
+      </h2>
+      <div className="mx-auto max-w-[66.5rem] mb-80">
+        <Accordion items={faqData} />
+      </div>
+    </div>
+  </section>
+);
```
Comment (GabeDiniz) [inline src/components/sections/FAQsection.tsx:29]: Remove the commented out code if its not needed

File: tailwind.config.mjs [hunk 2]
```diff
@@ -29,6 +29,11 @@ export default {
                 khakiYellow: '#F0E698',
                 dimGray: '#4A4F50',
             },
+            backgroundImage: {
+                'faq-image': "url('/src/assets/background/faq-background.png')",
+                
+
```
Comment (GabeDiniz) [inline tailwind.config.mjs:35]: Remove leading spaces

**Comment by juancwu** [comment]
Are the comments marked as resolved because they are being worked on @norachams ? If yes, a comment saying they are being worked on would be nice if changes have not been committed.

File: src/components/Accordion.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      {items.map((item, index) => (
+        <div key={index} className="px-3 w-full md:w-1/2 mb-5">
+          <div
+            className={`cursor-pointer flex justify-between items-center p-2 bg-white border-black rounded-xl border ${
+              activeIndex === index ? 'rounded-b-none' : ''
+            }`}
+            onClick={() => toggleAccordion(index)}
+          >
+            <h6 className="text-black">{item.question}</h6>
+            <span>{activeIndex === index ? '^' : '⌄'}</span>
```
Comment (juancwu) [inline src/components/Accordion.tsx:26]: I think using an proper icon would make it look better. There is a slight change in position and size when active.

I recommend using:
- https://github.com/tailwindlabs/heroicons?tab=readme-ov-file#react

A list of available icons:
- https://heroicons.com/

Comment (juancwu) [reply src/components/Accordion.tsx:26]: @norachams Just making sure this doesn't get forgotten.

**Review by juancwu — APPROVED** [review]

**Pull request merged**